hello-thirdparty/gpg-sdk/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+    sub module to download the Google Play games
+ */
+apply plugin: 'java'
```

Comment:
I need some plugin that has build targets so it can be a compile dependency in app.  java was easy :)  Do you know of  a better one?